tests/data_interface/test_get_data.py

```diff
@@ -169,7 +169,7 @@ def test_error_raised_for_reallllyyyy_bad_input_station_data(self):
             variable="Air Temperature at 2m",
             resolution="9 km",
             timescale="hourly",
-            data_type="Station",
+            data_type="Stations",
```

Comment:
@nicolejkeeney Very weird that is missed Station->Stations change in the tests did not fail with the current testing environment. I have made this change in the 1.2.0 release and will be merged this afternoon.

Reply:
Ahh I see what you were going for here. I think it is failing because it is returning a different message. I am looking into fixing the tests...

Reply:
Ok, fixed. Cleared for takeoff!